udata_front/theme/gouvfr/datagouv-components/.env

```diff
@@ -10,3 +10,6 @@ VITE_API_2_URL='https://www.data.gouv.fr/api/2/'
 VITE_SITE_ROOT_URL='https://www.data.gouv.fr/'
```

Comment:
Default value for `VITE_SCHEMA_VALIDATA_URL` should now be https://validata.fr

(not introduced in this PR, can't comment on the precise line)